.github/workflows/code-review.yml

```diff
@@ -2,11 +2,6 @@ name: AI Code Reviewer
 
 on:
   pull_request:
-    types:
-      - opened
-      - synchronize
-      - labeled
-      - unlabeled
     branches:
```

Comment:
You have added `branches` under `pull_request`, which means the workflow will only run when pull requests are made to these branches. If you want the workflow to run on pull requests to all branches, you should remove this.